ingestion/src/metadata/profiler/interface/nosql/profiler_interface.py

```diff
@@ -21,11 +21,10 @@
 
 from sqlalchemy import Column
 
-from metadata.generated.schema.entity.data.table import TableData
 from metadata.generated.schema.tests.customMetric import CustomMetric
 from metadata.profiler.adaptors.factory import factory
 from metadata.profiler.adaptors.nosql_adaptor import NoSQLAdaptor
-from metadata.profiler.api.models import ThreadPoolMetrics
+from metadata.profiler.api.models import SerializableTableData, ThreadPoolMetrics
```

Comment:
i am open to suggestions. but since `TableData` holds `any` in the schema, it needs to have some custom serialization logic so that it will be fail-proof. Another direction would be to hack it into the data model creation.

Reply:
It's a bit more verbose, but I think it'd be ok to just have a different class that does not inherit from TableData but is able to be casted into.

Something onlines of:

```python
class SerializableTableData(BaseModel):
    attr1: String
    attr2: Int
    attr3: Boolean

    def fromTableData(cls, TableData tableData) -> SerializableTableData:
        return cls(
            attr1=str(tableData.attr1),
            attr2=tableData.attr2,
            attr3=tableData.attr3
       )
```

I feel this should be safe enough so that if we modify TableData in a non-compatible way it should break. We could add some test to make sure of it as well

Reply:
we discussed a solution to change the TableData schema to `List[List[str]]`. It will be implemented in another PR.